capitals.rb

```diff
@@ -128,25 +128,51 @@
 }, {
     name: "Texas",
     capital: "Austin"
-}, {
-    name: "Utah",
-    capital: "Salt Lake City"
-}, {
-    name: "Vermont",
-    capital: "Montpelier"
-}, {
-    name: "Virginia",
-    capital: "Richmond"
-}, {
-    name: "Washington",
-    capital: "Olympia"
-}, {
-    name: "West Virginia",
-    capital: "Charleston"
-}, {
-    name: "Wisconsin",
-    capital: "Madison"
-}, {
-    name: "Wyoming",
-    capital: "Cheyenne"
+
+#}, {
+#    name: "Utah",
+#    capital: "Salt Lake City"
+#}, {
+#    name: "Vermont",
+#    capital: "Montpelier"
+#}, {
+#    name: "Virginia",
+#    capital: "Richmond"
+#}, {
+#    name: "Washington",
+#    capital: "Olympia"
+#}, {
+#    name: "West Virginia",
+#    capital: "Charleston"
+#}, {
+#    name: "Wisconsin",
+#    capital: "Madison"
+#}, {
+#    name: "Wyoming",
+#    capital: "Cheyenne"
 }]
+
+#shuffles states and capitals
+states = states.shuffle
+score = 0
+
```

Comment:
what you actually want to do is capture the score for each state, not all of them total. like this state[:correct]--but this would be after you inserted these keys in the hashes.